floodwatch/src/js/components/FilterParent.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import $ from 'jquery';
+import {GraphParent} from './GraphParent';
+import _ from 'lodash';
+import d3 from 'd3'
+
+type Props = {
+  side: string;
+  currentTopic: string;
+  updateMouseOver: Function;
```

Comment:
You can type the function too! So this would be `updateMouseOver: (topic: Topic) => void`